exporter/splunkhecexporter/exporter.go

```diff
@@ -54,22 +62,23 @@ type exporterOptions struct {
 func New(
 	config *Config,
 	logger *zap.Logger,
-) (component.MetricsExporterOld, error) {
+) (TraceAndMetricExporter, error) {
 
 	if config == nil {
-		return nil, errors.New("nil config")
+		return splunkExporter{}, errors.New("nil config")
```

Comment:
These are unusable as is, correct?  Curious why this change is preferred.

Reply:
Let me see if that change is actually needed. My go-fu is not that strong.

Reply:
OK that change wasn't needed.